client/driver/rkt_test.go

```diff
@@ -187,6 +188,18 @@ func TestRktDriver_Start_Wait(t *testing.T) {
 	case <-time.After(time.Duration(testutil.TestMultiplier()*15) * time.Second):
 		t.Fatalf("timeout")
 	}
+
+	// Make sure pod was removed #3561
+	var stderr bytes.Buffer
+	cmd := exec.Command(rktCmd, "status", handle.uuid)
+	cmd.Stdout = ioutil.Discard
+	cmd.Stderr = &stderr
+	if err := cmd.Run(); err == nil {
+		t.Fatalf("expected error running 'rkt status %s' on removed container", handle.uuid)
+	}
+	if out := stderr.String(); !strings.Contains(out, "no matches found") {
```

Comment:
This seems brittle if the returned error message ever changes, but other than checking for a non empty error message I don't have a better idea.

Reply:
Yeah, I hate resorting to `strings.Contains`, but if it ever fails due to a string change it should be a quick and easy fix (and caught by Travis, not a user at runtime).